### DISCUSSION TITLE
[Request] - On Error, show also the user file on top

### DISCUSSION THREAD

**Discussion by celentano99** [laravel/framework#53717] [Ideas]
When I write code, sometimes I get am error. To understand better, I need to look on what line was that, in the files updated by me. This is never the first file. I need to go in the logfile until I find the correct file ( sometimes I get until the end and I can find the file - when they are like 50 there ).

What I propose : on the error page, show above the current section, a summary section, that contains the "user" files ( the ones that are in the app/Controler/*, app/Model/*, app/Livewire/* and so on ) ( or a location that is not "by default" created by the framework). Like this, a new user can see what was wrong much easy. 

For the new users will be a big impact, they know what line that they wrote has causing the error.

**Comment by shaedrich** [comment]
Not sure if it's Laravel itself who can do something about that:
* Laravel uses [Flare/Ignition](https://flareapp.io/) by @spatie for their error page
* Spatie collapses "vendor frames" (files from your vendor folder) by default
* Spatie tries to also have "context" accompanying the error details
    * PHP version and config infos
    * request information
    * user if logged in
    * etc.
* etc.
